sinks/external.go

```diff
@@ -17,6 +17,7 @@ package sinks
 import (
```

Comment:
Actually, now that I read more of the code on the side it looks like the code is currently transitioning to `ExternalSink` as the interface. So yes, let's not rename ExternalSinkManager -> SinkManager. Sorry :) at least you have the separate commit!